chirp/drivers/wouxun.py

```diff
@@ -330,6 +330,64 @@ def get_features(self):
         rf.can_odd_split = True
         return rf
 
+    @property
+    def transmit_time_out_setting(self):
+        options = ["%s" % x for x in range(15, 615, 15)]
+        rs = RadioSetting(
+            "transmit_time_out",
+            "TX Time-out Timer",
+            RadioSettingValueList(
+                options,
+                current_index=self._memobj.settings.transmit_time_out))
+        return rs
+
+    @property
+    def transmit_time_out_alert_setting(self):
+        options = ["OFF"] + ["%s" % x for x in range(1, 11)]
+        rs = RadioSetting(
+            "tx_time_out_alert",
+            "TX Time-out Alert",
+            RadioSettingValueList(
+                options,
+                current_index=self._memobj.settings.tx_time_out_alert))
+        return rs
+
+    @property
+    def roger_beep_setting(self):
+        options = ["Off", "Begin", "End", "Both"]
+        rs = RadioSetting(
+            "roger_beep",
+            "Roger beep select",
+            RadioSettingValueList(
+                options,
+                current_index=self._memobj.settings.roger_beep))
+        return rs
+
+    @property
+    def power_save_setting(self):
+        rs = RadioSetting("power_save", "Power save",
+                          RadioSettingValueBoolean(
+                              self._memobj.settings.power_save))
+        return rs
+
+    @property
+    def vox_setting(self):
+        options = ["OFF"] + ["%s" % x for x in range(1, 11)]
+        rs = RadioSetting(
+            "vox",
+            "Vox",
+            RadioSettingValueList(
```

Comment:
Note: This was a `RadioSettingValueInteger` that went from 0 to 10. I changed it because the radio itself displays "OFF" at 0 and the `RadioSettingValueList` gives a better UX (drop-down choice rather than +/- to iterate through options).

Similar for `tx_time_out_alert`.